index.js

```diff
@@ -31,6 +31,7 @@ module.exports = {
     const sentryEnvironment = process.env.SENTRY_ENVIRONMENT || process.env.CONTEXT
     const sourceMapPath = inputs.sourceMapPath || PUBLISH_DIR
     const sourceMapUrlPrefix = inputs.sourceMapUrlPrefix || DEFAULT_SOURCE_MAP_URL_PREFIX
+    const releasePrefix = process.env.SENTRY_RELEASE_PREFIX || (inputs.releasePrefix || '')
```

Comment:
@robbiet480 Nit:  I don't think we need the parentheses here. Also, I think the input should take priority over the env variable. So it would be:
```
inputs.releasePrefix || process.env.SENTRY_RELEASE_PREFIX || ''
```

Reply:
Agreed, fixed!